AeroGearOAuth2/TrustedPersistantOAuth2Session.swift

```diff
@@ -135,13 +135,13 @@ public class KeychainWrap {
         keychainQuery[kSecAttrAccessible as String] = kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly
```

Comment:
This one need to be modified to kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly